source/application/AMDatamanAppController.cpp

```diff
@@ -242,10 +242,12 @@ bool AMDatamanAppController::startup() {
 			return AMErrorMon::errorAndReturn(this, AMDATAMANAPPCONTROLLER_STARTUP_ERROR_HANDING_NON_FIRST_TIME_USER, "Problem with Acquaman startup: handling non-first-time user.");
 	}
 
-
 	if(!startupRegisterDatabases())
 		return AMErrorMon::errorAndReturn(this, AMDATAMANAPPCONTROLLER_STARTUP_ERROR_REGISTERING_DATABASES, "Problem with Acquaman startup: registering databases.");
 
+	if(!startupDatabaseUpgrades())
```

Comment:
I'm assuming placing this method here covers both first time and every time cases?  Either way, it looks a little weird to be called here rather than after `startupCreateDatabases()`.